src/cdk/overlay/position/global-position-strategy.ts

```diff
@@ -86,14 +96,14 @@ export class GlobalPositionStrategy implements PositionStrategy {
   /**
    * Sets the overlay width and clears any previously set width.
    * @param value New width for the overlay
+   * @deprecated Pass the `width` through the `OverlayConfig`.
+   * @deletion-target 6.0.0
```

Comment:
This will have to be 7.0; any deprecated API has to stick around for at least one full major release cycle before being removed.

Reply:
Done.